recipe/meta.yaml

```diff
@@ -31,9 +33,10 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - libqdldl {{ version }}
+    - libqdldl 0.1.7
```

Comment:
when this gets updated next, someone might not know to update the pinning here. How about a comment or a jinja function that splits the version?

Reply:
Thanks, I added a comment